modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -58,8 +58,37 @@ The password is used only for bootstrapping, it is not saved anywhere.
 All future SSH sessions are authorized via a key/certificate pair.
 These parameters are configured in menu:Systems[Bootstrapping].
 
-You can also configure persistent parameters that are used system-wide, including the sudo user.
-For more information on configuring the sudo user, see xref:client-configuration:contact-methods-pushssh.adoc[].
+You can also configure persistent parameters that are used system-wide, including the sudo user to access the system as an unprivileged user instead of as root.
+
+
+
+.Procedure: Configuring Unprivileged SSH Access
```

Comment:
@vzhestkov I believe this needs to somehow point [here](https://www.uyuni-project.org/uyuni-docs/en/uyuni/specialized-guides/salt/salt-ssh.html#salt.ssh.user) since we're in the Push via Salt SSH context.

Reply:
Seems yes, but there is no config for the bundle in the example provided, it should be readjusted to what we have in Ubuntu section.

Reply:
I tried to improve it that way.  Maybe, more adjustments are needed.